<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-UGVL9IKA" failures="1" tests="6" name="C:/Users/Me/Desktop/SolomonsSeleniunProject/src/test/java/org/TestCases" time="16.746" errors="0" timestamp="2023-07-30T13:13:14 BST">
  <testcase classname="org.TestCases.LandingPageTest" name="landingPageRunner" time="0.394">
    <skipped/>
  </testcase> <!-- landingPageRunner -->
  <testcase classname="org.TestCases.footerTest" name="footerRunner" time="2.335"/>
  <testcase classname="org.TestCases.LoginTest" name="login" time="2.518"/>
  <testcase classname="org.TestCases.LandingPageTest" name="landingPageRunner" time="0.246">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at org.TestCases.LandingPageTest.landingPageRunner(LandingPageTest.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:263)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:62)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- landingPageRunner -->
  <testcase classname="org.TestCases.LoginTest" name="login" time="0.945"/>
  <testcase classname="org.TestCases.SubmenuTest" name="submenuRunner" time="10.944"/>
</testsuite> <!-- C:/Users/Me/Desktop/SolomonsSeleniunProject/src/test/java/org/TestCases -->
